xml/obs_basic_workflow.xml

```diff
@@ -616,8 +616,117 @@ Requires:         zool >= 1.5.6</screen>
+     (binaries) of a package, copying the build results into another
+     project.
+     Aggregating is used in the following situations:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       No changes in the source code are needed.
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       Packages are needed as a build requirement but are available
+       only in a different project.
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       Build results (binaries) of the aggregated package are copied
+       into the target's package repository.
+      </para>
+     </listitem>
+     <listitem>
+      <para>Package rebuilds are avoided.</para>
+     </listitem>
+    </itemizedlist>
+
+    <para>
+     The general signature of the <command>aggregatepac</command> command is:
+    </para>
+    <screen>&prompt.user;&osccmd; aggregatepac <replaceable
+     >SOURCEPRJ</replaceable> <replaceable
+      >SOURCEPAC</replaceable> <replaceable
+       >DESTPRJ</replaceable></screen>
+    <para>
+     For example, to aggregate the package <package>python-lxml</package>
+     from <uri>devel:language:python</uri> into your home project,
+     use the following command:
+    </para>
+    <screen>&prompt.user;&osccmd; aggregatepac devel:language:python python-lxml &obshome1;</screen>
+    <para>
+     The next time you update your working directory, the <filename>python-lxml</filename>
+     directory is downloaded with the <filename>_aggregate</filename> file.
+    </para>
+   </sect3>
+   <sect3 xml:id="sec.obs.basicworkflow.reuse.linking">
+    <title>Linking a Package</title>
+    <para>
+     Basically, a linked package is a clone of another package with some
+     additional modifications. Linking is used in the following situations:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       Some changes in the source code is needed, but it is impossible,
+       unpractical, or inconvenient to change it.
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       Separate the original source from own patches.
+      </para>
+     </listitem>
+    </itemizedlist>
+    <para>
+     The general signature of the <command>linkpac</command> command is:
+    </para>
+    <screen>&prompt.user;&osccmd; linkpac <replaceable
+     >SOURCEPRJ</replaceable> <replaceable
+      >SOURCEPAC</replaceable> <replaceable
+       >DESTPRJ</replaceable></screen>
+    <para>
+     For example, to link the package <package>python-lxml</package> from
+     <uri>devel:language:python</uri> into your home project, use the
+     following command:
+    </para>
+    <screen>&prompt.user;&osccmd; linkpac devel:language:python python-lxml &obshome1;</screen>
+    <para>
+     In contrast to aggregating, this time the checkout contains all the
+     files from the linked repository. To reduce it to a single file (like
+     in aggregating) you need to <quote>unexpand</quote> it in the working
```

Comment:
replace:
"like in" -> "like with"
"you need to" -> ", "

Reply:
Ok, done